superset-frontend/src/dashboard/components/HeaderActionsDropdown.jsx

```diff
@@ -137,6 +139,9 @@ class HeaderActionsDropdown extends React.PureComponent {
   }
 
   handleMenuClick({ key, domEvent }) {
+    const menu = document.querySelector(
```

Comment:
let's move this const into the specific download as image case, that way it's also covered by the comment showing the hackiness

Reply:
We can't declare variables in `case`, because it breaks an eslint rule `no-case-declarations`